File: android/src/main/java/com/valentingrigorean/arcgis_maps_flutter/utils/StringUtils.java
package com.valentingrigorean.arcgis_maps_flutter.utils;

public class StringUtils {
    public static boolean areEqual(String s1, String s2) {
        if (s1 == null && s2 == null)
            return true;
        if (s1 == null || s2 == null)
            return false;
        return s1.equals(s2);
    }
}

File: android/src/main/kotlin/com/valentingrigorean/arcgis_maps_flutter/utils/StringUtils.kt
package com.valentingrigorean.arcgis_maps_flutter.utils

object StringUtils {
    fun areEqual(s1: String?, s2: String?): Boolean {
        if (s1 == null && s2 == null) return true
        return if (s1 == null || s2 == null) false else s1 == s2
    }
}